WordPress/Tests/WP/DeprecatedParametersUnitTest.inc

```diff
@@ -0,0 +1,64 @@
+<?php
+
```

Comment:
> I have ordered the functions in the test in alphabetical order.

That wasn't my intent here - future functions, unless they start with `z`, will not be at the end. The grouped nature means that new functions added in the first group will naturally means all of the error line numbers below need to be updated anyway though.